docs/overrides/home.html

```diff
@@ -10,7 +10,7 @@
     <div class="px-1 ctx-header-text conteiner">
       
        <span class="shortcut-bar Space-Bd-BT"> 
-        <a class="shortcut-bar-text" href="direct/get-started/">START</a>&nbsp;&nbsp;&nbsp;
+        <a class="shortcut-bar-text" href="direct/get-started/" >START</a>&nbsp;&nbsp;&nbsp;
```

Comment:
Is adding this space intentional? What difference does it make?

Reply:
Oops. That was an artifact of attempting to add a hover tooltip and then deleting the tag.